code/datums/diseases/advance/symptoms/hug.dm

```diff
@@ -0,0 +1,45 @@
+/datum/symptom/hugging
+	name = "Hugging"
+	desc = "The virus causes the host to uncontrollably hug, pet, or otherwise nonharmfully interact with nearby creatures. This symptom only affects sapient creatures."
```

Comment:
what a stupid symptom

Reply:
wow

Reply:
It's for giving a semi-conditional mood buff for the crew, spreading the virus to people who have internals on (if your virus can spread via contact), and annoying people (if you meet the resistance 9 threshold). A symptom with that wide of a variety of effects deserves to be a level 8 symptom, IMO.

Reply:
Well, I'm removing the "annoying people" aspect, but the other two effects (spreading the virus and giving people mood buffs) sound like they're strong enough together for this to be a level 8 symptom that people might actually add to their viruses.